package.json

```diff
@@ -21,6 +21,7 @@
     "react-native-gesture-handler": "^1.1.0",
     "react-native-responsive-screen": "^1.2.2",
     "react-navigation": "^3.9.1",
+    "translate": "^1.1.0",
```

Comment:
Adding a library just for one method is a very very questionable thing. Explain to me why do we need it exactly?

Reply:
fixed